pkgs/by-name/kb/kbd/package.nix

```diff
@@ -13,10 +13,18 @@
   bzip2,
   xz,
   zstd,
+  bash,
   gitUpdater,
+  # This might be worthwhile to disable if the store lives on a compressed filesystem.
+  withCompression ? true,
+  # Allow disabling the unicode_start/unicode_stop utilities, which
+  # unfortunately pull in bash into the closure.
+  withScripts ? true,
```

Comment:
Maybe we can take the risk and disable these too by default? Disabling these in an overlay would require lots of rebuilds, and being able to use compression and the scripts, is something that would be required for specific applications, and a user that needs these features can build and use a modified version of `kbd` just for that.

Reply:
> Other distros (Debian) ship these scripts as well. I would only feel comfortable disabling these by default, if we find out who actually needs them.
> 
> Another option would be to rewrite them in C.
> 
> See the code for details what they do: [legionus/kbd@`master`/src/unicode_start#L18](https://github.com/legionus/kbd/blob/master/src/unicode_start?rgh-link-date=2025-05-27T12%3A08%3A50Z#L18)

OK I see. I have an idea :) How about using multiple outputs? The `bin` output shouldn't be installed by default IIRC.